src/vt/vrt/collection/balance/node_lb_data.cc

```diff
@@ -155,6 +165,21 @@ void NodeLBData::initialize() {
     theNodeLBData()->createLBDataFile();
   }
 #endif
+
+#if vt_check_enabled(ldms)
+  if (auto ldms_freq = getenv("VT_LDMS_MILLI_FREQ")) {
+    ldms_milli_freq_ = atoi(ldms_freq);
+  }
+  const auto xPtr = getenv("VT_LDMS_XPTR");
```

Comment:
Typo: name should match ldms function call bellow.
```suggestion
  const auto xPtr = getenv("VT_LDMS_XPRT");
```

Reply:
Fix variable name as well.